<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<<<<<<< Updated upstream
<testsuite ignored="0" hostname="DESKTOP-FGAD874" failures="0" tests="1" name="Default test" time="13.048" errors="0" timestamp="28 Jun 2021 06:58:52 GMT">
  <testcase classname="cucumber.api.testng.AbstractTestNGCucumberTests" name="feature" time="12.346"/>
=======
<testsuite hostname="DESKTOP-UN79FSK" ignored="0" name="Default test" tests="1" failures="1" timestamp="30 Jun 2021 12:54:29 GMT" time="0.525" errors="0">
  <testcase name="feature" time="0.217" classname="cucumber.api.testng.AbstractTestNGCucumberTests">
    <failure type="cucumber.runtime.CucumberException" message="java.lang.NullPointerException">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.NullPointerExceptionat cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:483)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)Caused by: java.lang.NullPointerExceptionat base.NewBaseClass.navigateToUrl(NewBaseClass.java:49)at base.NewBaseClass.openbrowser(NewBaseClass.java:70)at platformstepdefinition.PFQA_123_1.navigate_to_url_on_pfqa1231(PFQA_123_1.java:33)at âœ½.Given Navigate to URL on chrome, <fileName> PFQA_123_1(./src/test/java/platformfeatures/PFQA_123_1.feature:6)]]>
    </failure>
  </testcase> <!-- feature -->
>>>>>>> Stashed changes
</testsuite> <!-- Default test -->
